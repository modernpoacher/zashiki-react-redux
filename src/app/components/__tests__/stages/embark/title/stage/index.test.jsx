import React from 'react'
import renderer from 'react-test-renderer'

import Stage from '@modernpoacher/zashiki-react-redux/app/components/stages/embark/title/stage'

describe('@modernpoacher/zashiki-react-redux/app/components/stages/embark/title/stage', () => {
  describe('Always', () => {
    it('renders', () => {
      const component = (
        <Stage
          title='MOCK TITLE'
        />
      )

      expect(renderer.create(component).toJSON())
        .toMatchSnapshot()
    })
  })
})
