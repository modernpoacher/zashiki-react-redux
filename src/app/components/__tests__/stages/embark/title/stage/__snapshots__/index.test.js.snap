// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@modernpoacher/zashiki-react-redux/app/components/stages/embark/title/stage Always renders 1`] = `ShallowWrapper {}`;
